lib/migration.js

```diff
@@ -80,8 +84,33 @@ const fetchMigration = async (config) => {
     const filename = path.join(directory, `${timestamp++}-create-${entry.sys.id}-migration.${module ? 'cjs' : 'js'}`);
 
     const content = await generateMigrationScript(client, [entry]);
+    // Fetch migration script generated by contentful
+    let modifiedContent = content.toString();
+    // Modify migration script to use our helpers
+    const testDefaultValueRegex = /(defaultValue\({\s*)[^\:]+([^\)]+)/g;
+    if (testDefaultValueRegex.test(modifiedContent)) {
+      modifiedContent = content
+        .toString()
+        // Add call to utils.getDefaultLocale() to the top
+        .replace(
+          'module.exports = function (migration) {',`
```

Comment:
Diese Replacements werden sicher irgendwann nicht mehr funktionieren, wenn contentful etwas am Output ändert. Besonders, da hier nach einem festen String gesucht wird.
Es wäre gut wenn wir dafür zumindest einen Integration Test hätten, der läuft wenn wir ein release machen.

Reply:
Testing this is a bit hard as we would need to utilize the `generateMigrationScript` function from inside `contentful-cli` for this test to make sense which by itself would require a contentful space.

As long as we manually update the contentful-cli depencendy we can have an eye on this ;)